### PR TITLE
Quick review of Form login chapter

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: cookbook/security/form_login_setup.rst [hunk 1]
```diff
@@ -458,7 +462,8 @@ firewall matches *all* URLs, including ``/login_check``). If ``/login_check``
 doesn't match any firewall, you'll receive a ``Unable to find the controller
 for path "/login_check"`` exception.
 
-**4. Multiple firewalls don't share security context**
+4. Multiple Firewalls Don't share Security Context
```
Comment (WouterJ) [inline cookbook/security/form_login_setup.rst]: this and (5) seem to not be related to form login, but just general security pitfalls.

File: cookbook/security/form_login_setup.rst [hunk 2]
```diff
@@ -334,7 +336,8 @@ redirected to a 404 page instead of the login page, or that submitting
 the login form does nothing (you just see the login form over and over
 again).
 
-**2. Be sure the login page isn't secure (redirect loop!)**
+2. Be Sure the Login Page isn't Secure (Redirect Loop!)
```
Comment (xabbuh) [inline cookbook/security/form_login_setup.rst]: Isn't

File: cookbook/security/form_login_setup.rst (hunk 1 above)
Comment (xabbuh) [inline cookbook/security/form_login_setup.rst]: Share

File: cookbook/security/form_login_setup.rst [hunk 3]
```diff
@@ -467,7 +472,8 @@ to explicitly specify the same :ref:`reference-security-firewall-context`
 for different firewalls. But usually for most applications, having one
 main firewall is enough.
 
-**5. Routing error pages are not covered by firewalls**
+5. Routing Error Pages are not Covered by Firewalls
```
Comment (xabbuh) [inline cookbook/security/form_login_setup.rst]: Are

**Comment by xabbuh** [comment]
Thank you Wouter.

**Comment by xabbuh** [comment]
@WouterJ I tweaked one headline in 23f5bc8c37a285db80b69c3e242ab9c509c89d38 a bit to be consistent with how it is in the `2.6` branch. Let me know if you don't agree with that change.

**Pull request merged**